docs/guides/run-jobs-batch.mdx

```diff
@@ -23,12 +23,104 @@ Batch mode can shorten processing time if all jobs can be provided at the outset
 
 ![This diagram illustrates jobs submitted in a batch.  It shows five jobs, numbered 0 through 4, in a queue. The jobs are a mix of Estimator and Sampler.](/images/guides/batch.png 'Figure 1: Batch execution')
 
-There are multiple ways you can reconfigure your jobs to take advantage of the parallel processing provided by batching.
+## Open a batch
+
+You can open a runtime batch by using the context manager `with Batch(...)` or by initializing the `Batch`
+class. The batch starts when its first job begins execution.
+
+
+**Batch class**
+
+``` python
+from qiskit_ibm_runtime import Batch, Samplerv2 as Sampler, Estimatorv2 as Estimator
+
+backend = service.least_busy(operational=True, simulator=False)
+batch = Batch(service=service, backend=backend)
+estimator = Estimator(mode=batch)
+sampler = Sampler(mode=batch)
+batch.close()
+```
+
+
+**Context manager**
+
+The context manager automatically opens and closes the batch.
+
+``` python
+from qiskit_ibm_runtime import Batch, Samplerv2 as Sampler, Estimatorv2 as Estimator
+
+backend = service.least_busy(operational=True, simulator=False)
+with Batch(service=service, backend=backend):
+  estimator = Estimator()
+  sampler = Sampler()
+```
+
+When you start a batch,  you must specify a QPU. This can be done by specifying its name or by passing a `backend` object.
+
+**Specify a QPU by name**
+
+``` python
+service = QiskitRuntimeService()
+with Batch(service=service, backend="ibm_sherbrooke"):
+    ...
+```
+
+**Pass a `backend` object**
+
+``` python
+backend = service.least_busy(operational=True, simulator=False)
+with Batch(backend=backend):
+  ...
+```
+
+<span id="specify-batch-length"></span>
+## Batch length
+
+You can define the batch's maximum time to live (TTL) with the `max_time` parameter. This should exceed the longest job's execution time and be within the QPU's limits.
```

Comment:
But this page never talked about what maximum TTL is.

Reply:
> But this page never talked about what maximum TTL is.

Tried to clarify that in the recent commit